README.md

```diff
@@ -17,95 +18,110 @@ This module adds a new exec provider capable of executing PowerShell commands.
+All parameters are optional.
 
-* **command** - The actual PowerShell command to execute. Must either be fully qualified or a search path for the command must be provided. 
-* **creates** - The file to look for before running the command. The command will only run if the file doesn’t exist. *Note:* This parameter will not create a file, it will simply look for one.
-* **cwd** - The directory from which to run the command.
-* **environment** - Additional environment variables to set for a command. Multiple entries should be in an array.
-* **group** - The group to run the command as.  
-* **logoutput** - Whether to log command output in addition to logging the exit code. Valid values are 'true', 'false', and 'on_failure'. Defaults to 'on_failure, which only logs the output when the command has an exit code that does not match any value specified by the returns attribute. 
-* **onlyif** - Only run the exec if the command returns 0.
-* **path** - The search path used for command execution.
-* **refresh** - How to refresh the command.
-* **refreshonly** -  Used with `subscribe` and `notify` [metaparameters](http://docs.puppetlabs.com/references/latest/metaparameter.html) to refresh the command only when a dependent object is changed. Valid values are 'true' and 'false'.
-* **returns** - The expected return code(s). An error will be returned if the executed command returns something else. Defaults to 0. Can be specified as an array of acceptable return codes or a single value.
-* **timeout** - The maximum time the command should take.
-* **tries** - The number of times execution of the command should be tried. Defaults to '1'. 
-* **try_sleep** - The time to sleep in seconds between `tries`.
-* **umask** - The umask to be used while executing the command.
-* **unless** - The `exec` will run unless the command returns 0.
+#####`command`
+Specifies the actual PowerShell command to execute. Must either be fully qualified or a search path for the command must be provided. Valid options: String. Default: Undefined.
 
-##Limitations
+#####`environment`
+Sets additional environment variables to set for a command. Valid options: String, or an array of multiple options. Default: Undefined.
 
- * This module requires PowerShell to be installed and the `powershell.exe` to be available in the system PATH.
- * Only supported on Windows Server 2003 and above, and Windows 7 and 8
+#####`logoutput`
+Defines whether to log command output in addition to logging the exit code. If you specify 'on_failure', it only logs the output when the command has an exit code that does not match any value specified by the `returns` attribute. Valid options: 'true', 'false', and 'on_failure'. Default: 'on_failure'.
 
-##Development
+#####`onlyif`
+Runs the exec only if the command returns 0. Valid options: String. Default: Undefined.
+
+#####`path`
+Specifies the search path used for command execution. Valid options: String of the path, or a ; separated list. Default: Undefined. 
```

Comment:
If you've seen it as an array, and you'd like that listed as a valid option, could you please verify that it works?